tests/api/test_overdrive.py

```diff
@@ -1521,6 +1521,9 @@ def test_refresh_patron_access_token_is_fulfillment(self):
         self._default_collection.external_integration.setting(
             "overdrive_website_id"
         ).value = "100"
+        self._default_collection.external_integration.setting(
+            "overdrive_server_nickname"
+        ).value = "testing"
```

Comment:
@io7m It's not obvious why this needs to be set to `testing` in order for this test to pass. If it's not set to `testing`, it should default to the production ones.

Reply:
It looks like this is happening because of change in `OverdriveCoreAPI.__init__` (removal of ll. 255-258).